project_euler/problem_42/sol1.py

```diff
@@ -0,0 +1,44 @@
+"""
+The nth term of the sequence of triangle numbers is given by, tn = ½n(n+1); so the first ten triangle numbers are:
```

Comment:
Please wrap at 88 characters max as discussed in
https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md